packages/components/textarea/src/textarea.scss

```diff
@@ -1,9 +1,149 @@
+$sizes: md, lg;
+$variants: default, valid, invalid;
+$variant-selectors: ':not([valid],[invalid])', ':where([valid])', ':where([invalid])';
+$resize-types: none, both, horizontal, vertical, auto;
+
 :host {
-  --_border: 1px solid var(--_border-color);
-  --_border-color: #d3d3d3;
-  --_border-radius: 3px;
-  --_box-shadow: none;
-  --_gap: 0.25rem;
+  --_border-size: var(--sl-border-width-button-default);
+  --_border: var(--_border-size) solid var(--_border-color);
+  --_border-radius: var(--sl-border-radius-default);
+  --_box-shadow: var(--sl-box-shadow-none);
+  --_transition: 200ms ease-in-out;
+  --sl-textarea-min-height-md: 72px;
```

Comment:
I guess that there can be some cases in the future, where someone wants to use a smaller one, that's why I wanted to add here CSS custom properties, but I set as default the values (height) we currently have in Figma.

Reply:
So there is a new CSS unit we could use here: `lh` (`line-height`)? See https://caniuse.com/mdn-css_types_length_lh
And perhaps with a fallback for Firefox?

Reply:
I wanted to add `rows` attribute first, but when we use rows, the height is inconsistent between different browsers, that's why I added here these custom properties for the min-height.

So should I try with `lh` instead of pixels the in the custom properties? Or there will be also inconsistent height between Firefox and other browsers?

Reply:
I would try something like this: `--_min-height: calc(var(--sl-textarea-rows) * 1lh);`
And a fallback for FF using perhaps `@supports`???? Not sure how that would work exactly atm.

Reply:
Ok, I'll try this way :)